src/components/footer/HaukiFooter.tsx

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
```

Comment:
It seems that some of the styles of HaukiFooter.scss are now redundant.

Reply:
Good point, totally forgot them there!

- [ ] Remove redundant HaukiFooter.scss style definitions